templates/core/terraform/appgateway/locals.tf

```diff
@@ -24,4 +24,8 @@ locals {
   redirect_configuration_name        = "rdrcfg-tosecure"
 
   certificate_name = "cert-primary"
+  tre_core_tags = {
+    tre_id              = var.tre_id
+    tre_core_service_id = var.tre_id
+  }
```

Comment:
previously there was an 'environment' tag. which was removed. are we ok with that or we want to add it here?

Reply:
not sure, I think if was left there by a mistake.
@marrobi, WDYT?

Reply:
I can't see the removed tag here - shouldn't be anything with a fixed value, but we need to enable customers to add their own tags. Maybe that should be picked up as another user story?

Reply:
They can add their own tags via Azure portal / Azure API / TF, I left the ignore_changes in all resources.
Do you think we need a TRE API for that?

Reply:
templates/core/terraform/admin-jumpbox.tf line #65

Reply:
More, how do they add via TF - would be better by setting variables, rather than having their own fork with custom tags.

Reply:
lets create another separate story for that.